src/audit-log-api/src/use-cases/RetryMessageUseCase.ts

```diff
@@ -1,12 +1,13 @@
 import type { PromiseResult } from "shared-types"
-import { decodeBase64, logger } from "shared"
+import { logger } from "shared"
 import { isError } from "shared-types"
 
 import type GetLastFailedMessageEventUseCase from "./GetLastEventUseCase"
 import type RetrieveEventXmlFromS3 from "./RetrieveEventXmlFromS3UseCase"
 import type CreateRetryingEventUseCase from "./CreateRetryingEventUseCase"
 import type SendMessageToQueueUseCase from "./SendMessageToQueueUseCase"
 import validateEventForRetry from "./validateEventForRetry"
+import { decodeBase64 } from "shared"
```

Comment:
this was already imported on line 2, was this changed by your IDE?

Reply:
I think it changed while I was doing the refactoring. Removing `decodeBase64` and adding it back I guess. Good spot, I updated it.